Yosemite/Yosemite/Stores/CustomerStore.swift

```diff
@@ -6,6 +6,7 @@ public final class CustomerStore: Store {
 
     private let customerRemote: CustomerRemote
     private let searchRemote: WCAnalyticsCustomerRemote
+    private let sharedDerivedStorage: StorageType
```

Comment:
nit: This could also be initialized as a lazy var (rather than in the `init` method):

```suggestion
    private lazy var sharedDerivedStorage: StorageType = {
        return storageManager.writerDerivedStorage
    }()
```

Reply:
Thanks! I initially injected it to the initializer for testing, but as I made the methods private this was not necessary anymore. Changed to be lazy at 470010e1ff62dd8257c38cdc07f7242ec9933b83